uvicgse/project-virtual-team-2
# Fix #106 #118: Show/Search and Delete branches and tags from dropdown

Jamie implemented a feature on top of existing codebase to separate tags and branches. Made the search input field work and refactored both the design and code.


## Testing:
1. Navigate to the branches lists
2. observe that branches and tags are on two different lists
3. search for a branch and a tag
4. delete tags from the list

![59650007-6f6ba600-9139-11e9-90e7-e075441372a5](https://user-images.githubusercontent.com/39515277/59897517-d55a6680-93a1-11e9-9b3d-68cf0ca8332c.png)
